opentracing/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package opentracing provides an Appdash implementation of the OpenTracing
+// API.
+package opentracing
```

Comment:
Doesn't need to be a separate file, since it's so small. Let's place the package docs into `recorder.go`.